CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## [Unreleased]
 
+## [4.11.3]
```

Comment:
since we're still arguing about this, want to maybe remove this version and I'll just merge it as 'Unreleased'? if another thing actually triggers a release of 2019.8.x, we'll likely have to update and release this again, so we can just take these along if that happens.